calculator_silver.rb

```diff
@@ -0,0 +1,42 @@
+puts "Let us do some simple maths! What kind of math do you want to do?"
```

Comment:
It was a little confusing to have this second file in here. If you modify the other file and commit it, you'll still be able to see the previous version on GitHub.